include/lsst/afw/detection/python.h

```diff
@@ -45,11 +47,14 @@ namespace detection {
  * @see [pybind11 documentation](https://pybind11.readthedocs.io/en/stable/advanced/classes.html)
  */
 template <typename Base = Psf>
-class PyPsf : public lsst::afw::typehandling::StorableHelper<Base> {
+class PsfTrampoline : public StorableHelper<Base> {
 public:
-    using lsst::afw::typehandling::StorableHelper<Base>::StorableHelper;  // Inherit ctors
     using Image = typename Base::Image;
 
+    // delegate ctor to Base
```

Comment:
Since this is an unusual technique, I think you want full Doxygen documentation here. In particular, you should explicitly say that `args` are the constructor arguments for the `Psf` class being augmented.

Reply:
Added:

```
    /**
     * Delegating constructor for wrapped class.
     *
     * While we would like to simply inherit base class constructors, when doing so, we cannot
     * change their access specifiers.  One consequence is that it's not possible to use inheritance
     * to expose a protected constructor to python.  The alternative, used here, is to create a new
     * public constructor that delegates to the base class public or protected constructor with the
     * same signature.
     *
     * @tparam Args  Variadic type specification
     * @param ...args  Arguments to forward to the Base class constructor.
     */
```